src/wp-includes/media.php

```diff
@@ -1093,12 +1094,9 @@ function wp_get_attachment_image( $attachment_id, $size = 'thumbnail', $icon = f
 		$context = apply_filters( 'wp_get_attachment_image_context', 'wp_get_attachment_image' );
 		$attr    = wp_parse_args( $attr, $default_attr );
```

Comment:
See my comment above: Better set the two values in `$attr` here instead of modifying `$default_attr`.
```suggestion
		$context        = apply_filters( 'wp_get_attachment_image_context', 'wp_get_attachment_image' );
		$attr           = wp_parse_args( $attr, $default_attr );
		$attr['width']  = $width;
		$attr['height'] = $height;
```

Reply:
Good catch @felixarntz . Fixed in https://github.com/WordPress/wordpress-develop/pull/8758/commits/53776df91dcb72a8912a4a2daa0b5d34d73d8043